src/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,119 @@
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  CardMedia,
+  Collapse,
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import React, { useState } from 'react';
+import ReactPlayer from 'react-player';
+import { NavLink } from 'react-router-dom';
+import { save } from '../../services/localStorage/storage.js';
+import { ExpandMore } from './CourseCard.styled';
+
+const CourseCard = ({ data }) => {
+  const [hovered, setHovered] = useState(false);
+  const [expanded, setExpanded] = useState(false);
+
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  };
+
+  const { id, title, previewImageLink, lessonsCount, rating, meta } = data;
+  const { skills, courseVideoPreview } = meta;
+  const hasVideo = courseVideoPreview !== undefined;
+
+  const handleClick = () => {
+    save('lastCourseId', id);
+  };
+
+  return (
+    <Card
+      sx={{ height: '100%', display: 'flex', flexDirection: 'column' }}
+      onMouseEnter={() => setHovered(true)}
+      onMouseLeave={() => setHovered(false)}
+    >
+      {hasVideo && hovered ? (
+        <ReactPlayer
+          url={`https://cors-proxy.fringe.zone/${courseVideoPreview.link}`}
+          playing={hovered}
+          loop={hovered}
+          muted={true}
+          controls={false}
+          width="100%"
```

Comment:
Тут і далі стилі в одинарних кавичках
Always use double quotes (") for JSX attributes, but single quotes (') for all other JS. eslint: [jsx-quotes]
Why? Regular HTML attributes also typically use double quotes instead of single, so JSX attributes mirror this convention.

// bad
<Foo bar='bar' />

// good
<Foo bar="bar" />

// bad
<Foo style={{ left: "20px" }} />

// good
<Foo style={{ left: '20px' }} />

Reply:
> 
Винесла в змінну proxyURL, про quotes - не зрозуміла до кінця - в мене навіть prettier форматує зараз - в інлайнових стилях (style={{ display: 'block' }}) тільки одинарні кавички, а в таких пропсах (width="100%") - тільки подвійні очікуються.